Sources/App/Models/Build+Platform.swift

```diff
@@ -54,7 +54,7 @@ extension Build {
 
         /// Currently supported build platforms
         static var allActive: [Self] {
-            [.ios, .macosSpm, .macosXcodebuild, .tvos, .watchos]
+            [.ios, .macosSpm, .macosXcodebuild, .linux, .tvos, .watchos]
```

Comment:
This determines the platform order in the build matrix. I was thinking behind iOS and macOS but before tvOS & watchOS is a good place for it to appear.